packages/client/src/application/schema-settings-editable/types.ts

```
@@ -0,0 +1,88 @@
+import { ISchema } from '@tachybase/schema';
+
+export interface EditableSchemaSettingOptions<T = {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)